service/controller/aws/v22/resource_set.go

```diff
@@ -343,9 +343,13 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		certConfigResource,
 		clusterConfigMapResource,
 		kubeConfigResource,
-		appResource,
+
+		// Migration resources are for migrating from chartconfig to app CRs.
 		appMigrationResource,
 
+		// appResource is executed after migration resources.
+		appResource,
```

Comment:
This is prep for adding the configmap migration resource. These need to run before the app resource.